packages/ui/lib/Preview/index.js

```diff
@@ -1,15 +1,20 @@
+import React from "react";
 import MicromarkMd from "./MicromarkMd";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 const PreviewWrapper = styled.div`
   padding-left: 12px;
-  border-left: 4px solid #e2e8f0;
+  ${(p) =>
+    p.border &&
+    css`
+      border-left: 4px solid #e2e8f0;
+    `}
   min-height: 159px;
 `;
 
-function Preview({ content }) {
+function Preview({ content, border = true }) {
```

Comment:
to improve this, please refactor `border` to `borderd`
since border is a noun & bordered is adjective

Reply:
or, make it `borderless` and `false` by default since its more often to be bordered